pandas/core/index.py

```diff
@@ -1348,10 +1348,28 @@ def order(self, return_indexer=False, ascending=True):
             return sorted_index, _as
         else:
             return sorted_index
+    order = sort_values
 
     def sort(self, *args, **kwargs):
         raise TypeError('Cannot sort an %r object' % self.__class__.__name__)
```

Comment:
Can make this message: "cannot sort an Index object *in-place*, use sort_values instead"